ci/install-musl.sh

```diff
@@ -10,7 +10,7 @@ case ${1} in
         musl_version=1.2.5
         ;;
     *)
-        musl_version=1.1.24
+        test -z RUST_LIBC_UNSTABLE_MUSL_V1_2_3 && musl_version=1.1.24 || musl_version=1.2.3
```

Comment:
Is a `$` missing? (looks like it's checking the literal `RUST_LIBC_UNSTABLE_MUSL_V1_2_3`)

Reply:
Oops, this was a silly omission - will fix on next push